legacy/JavaScript/JavaScriptSDK.Module/AppInsightsModule.ts

```diff
@@ -64,7 +64,7 @@ class AppInsightsModule {
     }
 
     private static _download(aiConfig: Microsoft.ApplicationInsights.IConfig) {
-        AppInsightsModule.appInsightsInstance.config = aiConfig;
+        AppInsightsModule.getAppInsightsInstance().config = aiConfig;
```

Comment:
We probably won't ever deploy the stuff in the legacy folder again so I don't think we need these changes for the v1 SDK

Reply:
As legacy doesn't have it's own tsconfig.json, I needed to change this.
While I *could* put a tsconfig.json here, it's (I think) safer to fix this class -- just in case -- we need to publish the older version again.

Reply:
As discussed offline added a comment stating that this may not be published and therefore users should check their deployed version